src/utils.js

```diff
@@ -0,0 +1,25 @@
+function DiffDate(value) {
```

Comment:
util로 빼주신 부분 너무 좋습니다 👍 
다만 util 함수의 경우 소문자로 시작하는 경우가 일반적이라 diffDate 추천드립니다 (앞글자 대문자는 컴포넌트에)
그리고 해당 로직들 만드시는 것도 좋지만 date-fns 라는 라이브러리 이용하는 방법도 추천드려요!
```suggestion
function diffDate(value) {
```